frappe/utils/backups.py

```diff
@@ -336,7 +336,7 @@ def print_summary(self):
 
 		for _type, info in backup_summary.items():
 			template = f"{{0:{title}}}: {{1:{path}}} {{2}}"
-			print(template.format(_type.title(), info["path"], info["size"]))
+			print(template.format(_type.title(), os.path.join(os.getcwd(), info["path"]), info["size"]))
```

Comment:
ummm, use abspath? 


also idk for sure if we should do this. People parse output code :sweat_smile: Just check if FC code isn't affected by it at least.